corehq/apps/app_manager/templates/app_manager/partials/modules/case_tile_templates.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% if request|toggle_enabled:'CASE_LIST_TILE' %}
```

Comment:
nit: I think it'd be clearer to have the conditional in the `case_list.html` template around the `include` (I see the same thing done with `case_list_multi_select.html` a little farther up), but don't see anything wrong with this.

Reply:
Fair point, although I'm ambivalent. Leaving this as is.

Reply:
Would have agreed with Evan